.github/workflows/deploy.yml

```diff
@@ -282,7 +282,7 @@ jobs:
         uses: actions/upload-artifact@v4
         with:
           name: android-hybrid-sourcemap-artifact
-          path: /Users/runner/work/App/App/Mobile-Expensify/Android/build/generated/sourcemaps/react/release/index.android.bundle.map
+          path: /home/runner/work/App/App/Mobile-Expensify/Android/build/generated/sourcemaps/react/release/index.android.bundle.map
```

Comment:
This is a path difference between Linux and MacOS runners that I didn't realize.